### PR TITLE
Make modal version

### PR DESCRIPTION
**Pull request by Victoria129**
- Design a modal version using JavaScript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,13 +7,50 @@
...
+  <div class="pop-up-btn">
+    <button class="poppins-regular see-live-btn">See live<img class="live-icon" src="./assets/live-icon.png"/></button>
+    <button class="poppins-regular see-source-btn">See source<img id="github-modal-icon" src="./assets/github-icon.png"/></button>
+  </div>
+  </div>
+  </div>
+</div>
+</div>
+
+<!-- End of html for pop up -->
```
Comment (zieeco) [inline index.html]: - Hey great job with the pop however, according to the project requirement, you're to create the pop-up dynamically using javascript. please kindly use adjust your implementation to meet the project requirements

Comment (zieeco) [reply index.html]: - Also, you're to update the project section in the index.htm file by adding them dynamically using javascript. i.e, the HTML structures you have hard coded in the index.html file should no longer be there but created dynamically using javascript. Therefore, these [lines of codes](https://github.com/Victoria129/Portfolio/blob/4958428085c86e937af2aa167f0c40a00a28c42a/index.html#L99-L190) should be added dynamically. please kindly update this to meet the project requirement

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Victoria129,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js [hunk 2]
```diff
@@ -1,7 +1,99 @@
+const data = [
+  {
+    heading: "Tonic",
+    image: "./assets/project1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+    and typesetting industry. Lorem Ipsum has been the industry's 
+    standard dummy text ever since the 1500s, when an unknown 
+    printer took a galley of type and scrambled it to make a type 
+    specimen book. It has survived not only five centuries, but 
+    also the leap into electronic typesetting, remaining essent`,
```
Comment (tugsem) [inline app.js:10]: - [ ] You should add these two example attributes as you are gonna need them in your future job search. (You are going to use different languages and source links for each of your project )
```suggestion
   * technologies: ["Phaser3", "JavaScript", "Jest"],
   * link: "link_to_live_version.com",
```

File: style.css [hunk 3]
```diff
@@ -18,6 +18,92 @@ body {
...
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  background-color: #c1c7d0;
+}
```
Comment (tugsem) [inline style.css:34]: - [ ]  Please fix these spacing issues on your modal window as they highly effects user experience:
- On window width 375px, your buttons: 
image

- On window width 503px, your buttons: 
image
- on 1000px and more, the img goes bigger and card title and padding disappears: 
image

**Review by tugsem — APPROVED** [review]
## APPROVED 👍 
Hi @Victoria129 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ You implemented requested changes(Please look optional suggestions 👇 )
## Optional suggestions 🎯 
<h1> As the previous reviewer didn't address these although they are in the project requirements, please fix these issues under the review as they are important for your designing skills. </h1>

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**